src/postgraphql/cli.ts

```diff
@@ -31,13 +33,15 @@ program
   .option('-q, --graphql <path>', 'the route to mount the GraphQL server on. defaults to `/graphql`')
   .option('-i, --graphiql <path>', 'the route to mount the GraphiQL interface on. defaults to `/graphiql`')
   .option('-b, --disable-graphiql', 'disables the GraphiQL interface. overrides the GraphiQL route option')
-  .option('-e, --secret <string>', 'the secret to be used when creating and verifying JWTs. if none is provided auth will be disabled')
   .option('-t, --token <identifier>', 'the Postgres identifier for a composite type that will be used to create tokens')
   .option('-o, --cors', 'enable generous CORS settings. this is disabled by default, if possible use a proxy instead')
   .option('-a, --classic-ids', 'use classic global id field name. required to support Relay 1')
   .option('-j, --dynamic-json', 'enable dynamic JSON in GraphQL inputs and outputs. uses stringified JSON by default')
   .option('-M, --disable-default-mutations', 'disable default mutations, mutation will only be possible through Postgres functions')
   .option('-l, --body-size-limit <string>', 'set the maximum size of JSON bodies that can be parsed (default 100kB) The size can be given as a human-readable string, such as \'200kB\' or \'5MB\' (case insensitive).')
+  .option('--secret <string>', 'DEPRECATED: See jwt.secret')
+  .option('-e, --jwt.secret <string>', 'the secret to be used when creating and verifying JWTs. if none is provided auth will be disabled')
+  .option('-A, --jwt.audience <list>', 'a comma separated list of audiences your jwt token can contain. If no audience is given the audience defaults to `postgraphql`', assign('audience', list), jwtOptions)
```

Comment:
Could we build something a little more custom here for parsing? Like perhaps what we do for the `--schema` option:

```js
(option: string) => ({ audience: option.split(', ') })
```

It’s just a little shorter and we don’t have to define functions we may never re-use 😊